pydantic/fields.py

```diff
@@ -670,7 +675,9 @@ def Field(  # noqa: C901
     max_length: int | None = _Unset,
     **extra: Unpack[_EmptyKwargs],
 ) -> Any:
-    """Create a field for objects that can be configured.
+    """Usage docs: https://docs.pydantic.dev/dev-v2/usage/fields
```

Comment:
Why did you change some to 2.0 and some you added dev-v2?

Reply:
I think they are all `2.0`, should probably be changed to `2.1` if these changes are going to be released in the next minor release.

The point is we can't just use `latest` or `dev-v2` since it'll be wrong for someone reading 2.0.3 code in 6 months time, or 6 years time 🤯.